protocol/avdl/provision_ui.avdl

```diff
@@ -37,13 +37,13 @@ protocol provisionUi {
    SecretResponse should be returned by DisplayAndPromptSecret.  Use either secret or phrase.
    */
   record SecretResponse {
-    bytes secret;
-    string phrase;
+    union { null, bytes } secret;
```

Comment:
Making this match what the comment says. its not 100% the same, ie we could send both empty, but this allows at least one to be empty